Changelog.md

```diff
@@ -3,9 +3,13 @@
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
-and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
```

Comment:
Why remove the semver reference?

Reply:
I though it was silly, because there is no version number at this moment.
(which needs to be fixed too)